db-init/src/docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM flyway/flyway:7.5.4
-COPY database /flyway/sql
-COPY flyway.conf /flyway/conf
+COPY db-init/src/main/resources/database /flyway/sql
+COPY db-init/src/main/resources/flyway.conf /flyway/conf
```

Comment:
Temporary fix until LH SecRel sets the Docker build context to work with relative paths.

When this change was made in LH StarterKit: https://github.com/department-of-veterans-affairs/lighthouse-di-demo-java-starterkit/commit/eb6d84fd3cdc02ab2ac97875af086598b53514d3#diff-aae3933ca62acd95b8bb90f8643852c01d7bbdbd49e09ddcc54aed1a60099e04

Reply:
However this change breaks using `./gradlew docker`, which prevents developers from building locally and is instructed by LHDI at https://animated-carnival-57b3e7f5.pages.github.io/starterkits/java/development-guide/#setting-up-and-running-docker-cis-benchmark-tests-locally.